### PR TITLE
answer

### PR DESCRIPTION
**Pull request by IvanKolodiazhnyi**

### CODE REVIEW

**Comment by IvanKolodiazhnyi** [comment]
https://IvanKolodiazhnyi.github.io/layout_hello-world/
https://mate-academy.github.io/layout_task-guideline/

**Comment by IvanKolodiazhnyi** [comment]
https://IvanKolodiazhnyi.github.io/layout_hello-world/
https://IvanKolodiazhnyi.github.io/layout_hello-world/report/html_report/

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+    Hello, world!
+  </body>
+</html>
```
Comment (varseniuk) [inline src/index.html:12]: You need to add an empty line at the end of your code

**Review by varseniuk — CHANGES_REQUESTED** [review]
These links should be in the pull request description (not in the comments). You may also use markup language to make them prettier, like this:
`- [DEMO LINK](https://IvanKolodiazhnyi.github.io/layout_hello-world/)`

**Pull request closed without merging**